packages/graphql-yoga/__tests__/accept-header.spec.ts

```diff
@@ -202,6 +206,10 @@ describe('accept header', () => {
         accept: 'application/json',
       },
     })
-    expect(response.status).toEqual(406)
+    expect(response.status).toEqual(200)
```

Comment:
Same for this.
https://github.com/dotansimha/graphql-yoga/pull/1589/files#r944599273

Reply:
Same as above, when accepting `application/json` the server should not use 4xx and 5xx ([see spec](https://graphql.github.io/graphql-over-http/draft/#sec-application-json)).